src/components/Header.jsx

```diff
@@ -0,0 +1,10 @@
+import { Link } from "react-router-dom";
```

Comment:
I think this is the homepage? This is where you would want to present a list of all articles so consider which element the path of "/" is rendering.